packages/crayons-core/src/components/button/button.scss

```diff
@@ -7,27 +7,75 @@
   @prop --fw-button-font-weight: The font weight of the button.
   @prop --fw-button-font-size: The font size of the normal button.
   @prop --fw-button-small-font-size: The font size of the small button.
+  @prop --fw-button-normal-line-height: Line height of the normal button.
+  @prop --fw-button-small-line-height: Line height of the small button.
+  @prop --fw-button-primary-border: The border color of the primary button.
+  @prop --fw-button-primary-border-active: The border color of the primary button on active state.
+  @prop --fw-button-primary-bg: The background color of the primary button.
+  @prop --fw-button-primary-bg-dark: The background color of the primary dark button.
+  @prop --fw-button-primary-color: The text color of the primary button.
+  @prop --fw-button-primary-disabled-color: The text color of the primary disabled button.
+  @prop --fw-button-primary-disabled-border: The border color of the primary disabled button.
+  @prop --fw-button-primary-disabled-bg: The background color of the primary disabled button.
+  @prop --fw-button-primary-disabled-bg-dark: The background color of the primary disabled dark button.
+  @prop --fw-button-secondary-border: The border color of the secondary button.
+  @prop --fw-button-secondary-border-active: The border color of the secondary button on active state.
+  @prop --fw-button-secondary-bg: The background color of the secondary button.
+  @prop --fw-button-secondary-bg-dark: The background color of the secondary dark button.
+  @prop --fw-button-secondary-color: The text color of the secondary button.
+  @prop --fw-button-secondary-disabled-color: The text color of the secondary disabled button.
+  @prop --fw-button-secondary-disabled-border: The border color of the secondary disabled button.
+  @prop --fw-button-secondary-disabled-bg: The background color of the secondary disabled button.
+  @prop --fw-button-active-box-shadow: Box shadow of active button.
+  @prop --fw-spinner-color-button-secondary: Spinner color of the secondary button.
```

Comment:
Can`--fw-spinner-color-button-secondary` be renamed to `--fw-button-secondary-spinner-color`

Reply:
Fixed.